crates/beerus-core/src/lightclient/beerus.rs

```diff
@@ -282,35 +282,6 @@ impl BeerusLightClient {
         &self.sync_status
     }
 
-    /// Get the storage at a given address/key.
-    /// This function is used to get the storage at a given address and key.
-    ///
-    /// # Arguments
-    ///
-    /// * `contract_address` - The StarkNet contract address.
-    /// * `storage_key` - The storage key.
-    ///
-    /// # Returns
-    ///
-    /// `Ok(FieldElement)` if the operation was successful.
-    /// `Err(eyre::Report)` if the operation failed.
-    pub async fn starknet_get_storage_at(
```

Comment:
This file contains all the methods that interact with StarkNet and Ethereum. However, the `get_storage_at` method now only interacts with StarkNet. It makes sense to remove it from here and use the one from `starknet_lightclient`. I  also remove the integration tests from `beerus_core` since they don't make sense anymore.
@phklive  what do you think?

Reply:
We need to keep using this one, using the last block from ethereum makes sure that the data that we return is valid and safe. 

If not a starknet node could lie considering that the data would not be validated on L1 yet.